html/modules/custom/hr_paragraphs/hr_paragraphs.module

```diff
@@ -522,6 +522,33 @@ function hr_paragraphs_preprocess_paragraph(&$variables) {
       '#suffix' => '</div>',
     ];
   }
+  else {
+    if ($paragraph->hasField('field_text') && isset($variables['content']['field_text'][0]['#text'])) {
+      $local_domains = [
+        'response.reliefweb.int',
+      ];
```

Comment:
It feels like this array is redundant since the config includes the domain allow-list. Should we be pulling from field settings to get this, or leave it hardcoded?

Reply:
Plan B, https://humanitarian.atlassian.net/browse/RW-849

Reply:
https://github.com/UN-OCHA/ocha_link_fixer